polaris.shopify.com/src/components/Page/Page.module.scss

```diff
@@ -59,15 +59,18 @@
 }
 
 .TOCWrapper {
+  @include layout-scrollbars();
+
   align-self: flex-start;
   position: sticky;
   top: calc(var(--header-height) + 4.25rem);
-  padding: 0 0 0 1.5rem;
+  padding: var(--p-space-4) var(--p-space-2) var(--p-space-2);
   width: var(--toc-width);
   overscroll-behavior: contain;
   overflow: auto;
-  box-shadow: inset 1px 0 var(--border-color);
-  @include layout-scrollbars();
+  border: 1.5px solid rgba(0, 0, 0, 0.2);
+  border-radius: var(--p-border-radius-3);
+  background-color: rgba(241, 241, 241, 0.2);
```

Comment:
Background color on Figma is 20% opacity of parent frame (which uses `bg-app` token aka `rgb(241, 241, 241)`). To keep scope of this PR constrained to the TOC, I added the background color here but we don't have a color token that adds the 20% opacity so I ended up using the rgba color instead.